pkg/utils/utils.go

```diff
@@ -83,3 +84,13 @@ func GetNetNames(pciAddr string) ([]string, error) {
 
 	return names, nil
 }
+
+// GetNetDevDriver returns current driver attached to a pci device from its pci address
+func GetNetDevDriver(pciAddr string) (string, error) {
```

Comment:
it will work for any pci device right ? not just netdev

perhaps rename to: `GetPCIDevDriver`